.drone.yml

```diff
@@ -230,3 +230,34 @@ steps:
       VERSION: "${DRONE_TAG}"
     depends_on:
       - clone
+
+---
+kind: pipeline
+type: kubernetes
+name: migration deploy tag
+trigger:
+  event:
+    - tag
+  branch:
+    - main
+depends_on:
+  - build tag
+
+steps:
+  - name: deploy migrator to cs-dev
+    image: quay.io/ukhomeofficedigital/kd
+    commands:
+      - cd kube
+      - ./deploy.sh
+    environment:
+      ENVIRONMENT: cs-dev
+      DEPLOYMENT_TYPE: migrator
+      SQS_SECRET_NAME: case-migrator-sqs
+      CASE_CREATOR_PORT: 8080
+      KUBE_TOKEN:
+        from_secret: hocs_case_creator_cs_dev
+      KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+      CLUSTER_NAME: acp-notprod
+      VERSION: "${DRONE_TAG}"
+    depends_on:
+      - clone
```

Comment:
You should set up your text editor to end lines with the line terminator sequence. It'll play better with `diff`.